docs/01_introduction.md

```diff
@@ -55,23 +53,21 @@ consistent testing of incoming contributions, and thorough regression testing be
 EasyBuild is ***not*** **YABT (Yet Another Build Tool)**: it does not replace established build
 tools like CMake or ``make``, it wraps around them.
 If the installation procedure of a software package involves running some unholy trinity of tools while whispering
-the correct magic incantations, EasyBuild just automates this process so you don't have to go through it manually.
+the correct magic incantations, EasyBuild seeks to automate this process for you.
```

Comment:
maybe just `EasyBuild automates this process`?

Reply:
Yes!